### ISSUE TITLE
Shift APIs to nodejs from Firebase functions.

### ISSUE THREAD

**Issue by nil1729** [nil1729/social_media_noob#2]
From next year firebase functions not offer any testing or executions without a `Blaze Plan`. I need to shift all Apis to my own node js server and for verifying users we can use `Firebase Admin SDK`. 
So for this project working, I need to create Apis.

Like [Vue-Phone-Store](https://github.com/nil1729/vue-phone-store)

**Comment by asadkhalid305** [comment]
Hey, this seems interesting. Can I work on this? Please assign.

**Comment by nil1729** [comment]
Yeah. Go on. I am also working with you because there have a lot of APIs we need to configure. 
I assign it to you.
Thanks.

**Comment by nil1729** [comment]
If you already started working on this project. Please create a separate branch for NodeJS and create PR from that branch. It will be easier to work with.

**Comment by asadkhalid305** [comment]
> If you already started working on this project. Please create a separate branch for NodeJS and create PR from that branch. It will be easier to work with.

Ok. I will start working on this from tomorrow. Can we use some kind of messaging app for in depth discussion?

**Comment by nil1729** [comment]
Yeah No problem. 
I am on discord. You can ping me on discord or my email. I will share you my contact details here.

Thanks😇

**Comment by mohnishbaker** [comment]
Hi Can I please work on this ? I am new to open source contributions and I could really use some work as well as some help.

**Comment by nil1729** [comment]
Yeah. Go on

**Comment by LordNayan** [comment]
Hello, I would like to help to migrate the apis. Please let me know the specifics over mail nayan.lakhwani123@gmail.com

**Comment by nil1729** [comment]
Thank you so much. Can we discuss some other streaming services using nodejs and device detection and optimizing the APIs? I want to learn these techs very severely.
